docs/architecture/session.md

```diff
@@ -0,0 +1,66 @@
+# Session
+
+__Session__ is the actor that encapsulates the logic of the sessions 
+associated to each TCP connections.
+
+
+
+## Actor creation and registration
+
+The creation of the session actor and its registration into the system registry are
```

Comment:
The creation of the session actor and its addition to the system registry are
performed by the `connection_manager` process: